src/pages/files/FileIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactElement } from 'react'
+import { File } from 'react-feather'
+import { StripedWrapper } from '../../components/StripedWrapper'
+
+export function FileIcon(): ReactElement {
+  return (
+    <StripedWrapper>
+      <File />
```

Comment:
maybe this could be prop and then the `FileIcon.tsx` and `FolderIcon.tsx` could be merged?